### PR TITLE
Week 4: add markers using data loaded from CSV and style.css fix

### PR DESCRIPTION
**Pull request by raniarusly**
image

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -20,19 +21,29 @@ h2 {
 h3 {
   font-family: 'Sora', Sans-serif;
   font-weight:600;
-  font-size: 20px;
+  font-size: 22px;
   line-height: 80%;
+  text-decoration: none!important;
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/style.css:26]:
```suggestion
  text-decoration: none !important;
```

File: portfolio/src/main/webapp/WEB-INF/city-data.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Florence,43.766667,11.25,June 2019
+Budapest,47.5,19.083333,April 2019
+Santorini,36.3932,25.4615,July 2019
+Seoul,37.5985,126.9783,December 2019
+Prague,50.0755,14.4378,March 2019
+Bali,-8.3405,115.0920,May 2018
+Paris,48.8566,2.3522,September 2018
+Surabaya,-7.2575,112.7521,June 2020
+Bandung,-6.9175,107.6191,June 2020
+Edinburgh,55.9533,-3.1883,February 2020
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/WEB-INF/city-data.csv:12]: I don't think this file should be in `WEB-INF`, I'd move it to the server code directory as it's only used there under a directory called `resources` maybe?

From https://docs.oracle.com/cd/E13222_01/wls/docs90/webapp/configurewebapp.html
> The WEB-INF directory contains the deployment descriptors for the Web application (web.xml and weblogic.xml) and two subdirectories for storing compiled Java classes and library JAR files. These subdirectories are respectively named classes and lib. JSP taglibs are stored in the WEB-INF directory at the top level of the staging directory. The Java classes include servlets, helper classes and, if desired, precompiled JSPs.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -158,33 +158,37 @@ async function checkLogin(){
...
-  const jakarta = {lat: -6.175540, lng: 106.82743};
-
-  const map = new google.maps.Map( document.getElementById("map"), {
-    center: jakarta,
-    zoom: 15,
-    mapTypeControlOptions: { mapTypeIds: ['roadmap', 'satellite', 'hybrid', 'terrain', 'night_map'] }
+  fetch("/city-data").then(result => result.json()).then((cities) => {
+      console.log(cities);
+      const map = new google.maps.Map( document.getElementById("map"), {
+        center: {lat: -6.175540, lng: 106.82743},
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:164]: nit: please add an inline comment or leave it as a named constant. just to know what are these numbers for :)

File: portfolio/src/main/webapp/WEB-INF/city-data.csv (hunk 2 above)
Comment (raniarusly) [reply portfolio/src/main/webapp/WEB-INF/city-data.csv:12]: Thanks Mostafa for the feedback! May I know that the server code directory is, is it under "main" or "webapp"?
I was following the directory structure from the example
Screenshot 2020-07-30 at 8 18 54 AM

Comment (MostafaGabriel) [reply portfolio/src/main/webapp/WEB-INF/city-data.csv:12]: Thanks Rania! I don't know why it's added there in the example but looks like this is not the best practice :)
The server code is the one under `portfolio/src/main/java/com/google/sps/` (i.e. where you handle the requests and where you also read the file), anything under `portfolio/src/main/webapp/` is called the client code (i.e. where you call the server and have the UI components)

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -158,11 +158,13 @@ async function checkLogin(){
 
 /* Creates a map and adds it to the page. */
 function initMap() {
+  const JAKARTA = {lat: -6.175540, lng: 106.82743};
+  const ZOOM_SCALE = 5;
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:162]: nit: please use `lowerCamelCase` for constants in a function scope. [ref](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names)

File: portfolio/src/main/webapp/WEB-INF/city-data.csv (hunk 2 above)
Comment (raniarusly) [reply portfolio/src/main/webapp/WEB-INF/city-data.csv:12]: Got it, thank you Mostafa, will make the change now :D

**Pull request merged**